ruleset/mutable.md

```diff
@@ -73,4 +73,8 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 
 If the rules are changed so that further play is impossible, or if the legality of a move cannot be determined with finality, or if by the Judge's best reasoning, not overruled, a move appears equally legal and illegal, then the first player unable to complete a turn is the winner.
 
-This rule takes precedence over every other rule determining the winner.
\ No newline at end of file
+This rule takes precedence over every other rule determining the winner.
+
+## 214
+
+Players should not be able to throw dice during their turn
```

Comment:
I'm concerned about the wording of this rule. This rule seems to be targeting the mechanism by which points are decided rather than directly targeting the primary concern: the accrual and allocation of points. "Players should not be able to", but what if they are; do the points get awarded? Also, the online version seems to indicate there is a different way of awarding points that doesn't include a dice.

I would feel more comfortable if this rule specifically stated players do not accrue points. Furthermore, I think there should be an arbitrary end-condition (like a date or number of rounds) at which point this rule is no longer in affect. At that point, we can then resubmit the rule to the group to be ratified for a further duration. The reason for this end-of-life condition is because this rule has a lot of power of the remainder of the game; to get this rule repealed could take a significant amount of negotiation and or bargaining. With the End-Of-Life condition, there is no bargaining; it simply stops being enacted. Thoughts?